fs/layer/node.go

```diff
@@ -187,15 +187,7 @@ func (f *FuseOperationCounter) Run(ctx context.Context) {
 	}
 }
 
-func incFuseOpFailureMetric(operationName string, layer digest.Digest) {
-	metric, ok := fuseOpFailureMetrics[operationName]
-	if !ok {
-		metric = commonmetrics.FuseUnknownFailureCount
-	}
-	commonmetrics.IncOperationCount(metric, layer)
-}
-
-// logFSOperations may cause sensitive information to be emitted to logs
+// logAndIncrementOpCounter may cause sensitive information to be emitted to logs
```

Comment:
Can you take a look here? The comment change does not appear to be intentional.

Reply:
Oops, fixed.